builtin_interfaces/Time t0_pub_send
builtin_interfaces/Time t1_sub_recv
builtin_interfaces/Time t2_sub_send
uint32 seq
uint32 payload_size_bytes
uint8[] payload

